### PR TITLE
release 4.0.0

### PR DESCRIPTION
**Pull request by Lightning-**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,43 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v4.0.0](https://github.com/voxpupuli/puppet-windows_power/tree/v4.0.0) (2024-05-10)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-windows_power/compare/v3.0.2...v4.0.0)
+
+This is a complete module rewrite/modernization (almost from scratch, but being inspired by what has been there before).
```
Comment (bastelfreak) [inline CHANGELOG.md:11]: nice release-summary :+1:

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,43 @@ All notable changes to this project will be documented in this file.
...
+
+**Closed issues:**
+
+- Settings can only be changed for the currently active scheme [\#36](https://github.com/voxpupuli/puppet-windows_power/issues/36)
+- Use in-built Windows Facter facts [\#9](https://github.com/voxpupuli/puppet-windows_power/issues/9)
+
+**Merged pull requests:**
+
+- drop stdlib dependency [\#53](https://github.com/voxpupuli/puppet-windows_power/pull/53) ([Lightning-](https://github.com/Lightning-))
+- Allow stdlib 8.0.0 [\#48](https://github.com/voxpupuli/puppet-windows_power/pull/48) ([smortex](https://github.com/smortex))
```
Comment (bastelfreak) [inline CHANGELOG.md:36]: now that stdlib got dropped as a dependency, I think we should add skip-changelog to this PR

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,43 @@ All notable changes to this project will be documented in this file.
...
+- modulesync 3.0.0 & puppet-lint updates [\#45](https://github.com/voxpupuli/puppet-windows_power/pull/45) ([bastelfreak](https://github.com/bastelfreak))
+- Allow puppetlabs/stdlib 6.x [\#42](https://github.com/voxpupuli/puppet-windows_power/pull/42) ([dhoppe](https://github.com/dhoppe))
```
Comment (bastelfreak) [inline CHANGELOG.md:38]: same here

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,43 @@ All notable changes to this project will be documented in this file.
...
+- Add example for changing power scheme settings [\#39](https://github.com/voxpupuli/puppet-windows_power/pull/39) ([thoscut](https://github.com/thoscut))
+- Remove duplicate CONTRIBUTING.md file [\#38](https://github.com/voxpupuli/puppet-windows_power/pull/38) ([dhoppe](https://github.com/dhoppe))
+- fixed all the execs, ps command failed and check was bad [\#35](https://github.com/voxpupuli/puppet-windows_power/pull/35) ([Nekototori](https://github.com/Nekototori))
```
Comment (bastelfreak) [inline CHANGELOG.md:41]: should this get a `bug` label?

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,43 @@ All notable changes to this project will be documented in this file.
...
+- fixed resource dependency and added testing to catch in the future [\#34](https://github.com/voxpupuli/puppet-windows_power/pull/34) ([Nekototori](https://github.com/Nekototori))
```
Comment (bastelfreak) [inline CHANGELOG.md:42]: sounds also like a bug?

**Comment by Lightning-** [comment]
was spreading some labels and rebased changelog: now this should reflect the rewrite in a more sane way

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**